app/assets/stylesheets/dummy/base/_typography.scss

```diff
@@ -0,0 +1,16 @@
+
+
+//cadenas como valores css
```

Comment:
if you add commented code, it's better to remove it from the repo, in the end, Git never forgets